mono/metadata/appdomain.c

```diff
@@ -1898,6 +1898,30 @@ real_load (gchar **search_path, const gchar *culture, const gchar *name, gboolea
 	return result;
 }
 
+//Keep it ordered
+static const char *priority_assemblies[] = {
+	"System.Runtime.InteropServices.RuntimeInformation",
+};
```

Comment:
Here is a list of additional required assemblies

System.Globalization.Extensions.dll
System.IO.Compression.dll
System.Net.Http.dll
System.Text.Encoding.CodePages.dll